literals/numbers.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/binary"
 	"go/ast"
 	"go/token"
+	"math"
 )
 
 func getBoundsCheck(pos string) *ast.AssignStmt {
```

Comment:
What's the point of this check? At the obfuscation stage it is guaranteed that the size of the array of the necessary size.

Reply:
It's a performance optimization to remove unessecary bounds checks. See https://docs.google.com/document/u/0/d/1vdAEAjYdzjnPA9WDOQ1e4e05cYVMpqSxJYZT33Cqw2g.